src/uid/UniqueId.java

```diff
@@ -249,6 +251,17 @@ public void dropCaches() {
   }
 
   /**
+   * Due to the var cache_hits type is int, but the max of int is 2147483648, and int happen spilling
+   */
+  private void reNumCache() {
```

Comment:
I'd rather just change the hits to a long instead of rolling over this low.

Reply:
hello @manolama, Please pay attention to the tsd.uid.ids-available is minus #1123, How to fix the ids-available that is positive? 
thanks!